.github/workflows/test.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.20.x, 1.21.x]
+        go-version: [1.20.x, 1.21.x, 1.22.x, 1.23.x]
```

Comment:
@s1061123  do I need to remove older version and keep same as other file to keep two go versions only